docs/examples.md

```diff
@@ -1,5 +1,92 @@
 # Examples
 
+## Browser &lt;script&gt; tag
+```html
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+</head>
+<body>
+  <script src="aepp-sdk.browser.js"></script>
+  <script type="text/javascript">
+    Ae.default.create('https://sdk-testnet.aepps.com').then(client => {
+      client.height().then(height => {
+        console.log('Current Block', height)
+      })
+    })
+  </script>
+</body>
+</html>
+```
+
+## nodejs (without compiling)
+
+Use either yarn or npm to install the sdk.
+Use a prebuilt package from npmjs
+```
+yarn add @aeternity/aepp-sdk
+```
+Or use a branch directly from github
+```
+yarn add aeternity/aepp-sdk-js#feature/sweet-api
+```
+create index.js
+```js
+const Ae = require('@aeternity/aepp-sdk')
```

Comment:
If we compile async/await, we can also assume module syntax can be used, so
`import Ae from '@aeternity/aepp-sdk' seems to be more coherent.